governance.md

```diff
@@ -70,8 +70,10 @@ Duties
 
 Qualifications
 
-* Nominated by community
-* Appointed by Steering Committee
+* Nominated and Appointed by Steering Committee
+  * 2 year terms, term limit of consecutive 4 years (after, individual must take off 1 year)
```

Comment:
```suggestion
  * 2 year terms, term limit of consecutive 4 years (after, individual must take 1 year off)
```

Reply:
Also, for clarification, does this mean that after a 1yr break, a user can come back and serve for another consecutive 4 years?

Reply:
Yes, I think so.